cli/src/index.js

```diff
@@ -233,4 +237,4 @@ for (const [key, value] of Object.entries(encoders)) {
   );
 }
 
-program.parse(process.argv);
+await program.parseAsync(process.argv);
```

Comment:
Note that we are using top level await here, which is required to prevent uncaught promise rejection errors. Currently, the CLI specifies `^12.20.2` which I don't think ships it, but the other two listed versions do support it.